problems/sumEvenNumbers.js

```diff
@@ -1,16 +1,22 @@
 /** Sum all even numbers in an array
- * 
-* @param {number[]} arr - The input array
-* @returns {number} - The sum of all even numbers
-*
-* ex: sumEvenNumbers([1, 2, 3, 4])
-* returns: 6 (because 2 + 4 = 6)
-*
-* ex: sumEvenNumbers([4, 3, 10])
-* returns: 14
-*/
-function sumEvenNumbers() {
+ *
+ * @param {number[]} arr - The input array
+ * @returns {number} - The sum of all even numbers
+ *
+ * ex: sumEvenNumbers([1, 2, 3, 4])
+ * returns: 6 (because 2 + 4 = 6)
+ *
+ * ex: sumEvenNumbers([4, 3, 10])
+ * returns: 14
+ */
+const sumEvenNumbers = (arr) => {
+  let sum = 0;
+  for (let i = 0; i <= arr.length; i++) {
```

Comment:
We don't want `i` to every go up to the array length `i <= arr.length`. Remember that the last index of an array is always `array.length - 1`